src/playground/widget-editor/src/components/editor-form/component.js

```diff
@@ -80,6 +80,18 @@ const EditorForm = () => {
           options={widgets || []}
         />
       </div>
+      <div className="input">
+        <label htmlFor="area-intersection">Area intersection ID</label>
```

Comment:
`htmlFor` always references the `id` of the input. Your input below doesn't have an `id` attribute so this form is not accessible.

You don't need to make a change now. I'm just putting this here as general feedback.

Reply:
this is on the playground so probably won't care about accessibility here as of now